github/actions/client.go

```diff
@@ -587,24 +567,18 @@ func (c *Client) doSessionRequest(ctx context.Context, method, url string, reque
 }
 
 func (c *Client) AcquireJobs(ctx context.Context, runnerScaleSetId int, messageQueueAccessToken string, requestIds []int64) ([]int64, error) {
-	u := fmt.Sprintf("%s/%s/%d/acquirejobs?api-version=6.0-preview", *c.ActionsServiceURL, scaleSetEndpoint, runnerScaleSetId)
+	path := fmt.Sprintf("/%s/%d/acquirejobs", scaleSetEndpoint, runnerScaleSetId)
 
 	body, err := json.Marshal(requestIds)
 	if err != nil {
 		return nil, err
 	}
 
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, u, bytes.NewBuffer(body))
+	req, err := c.NewActionsServiceRequest(ctx, http.MethodPost, path, bytes.NewBuffer(body))
```

Comment:
`http.NewRequestWithContext` since we are passing the `messageQueueAccessToken` token?

Reply:
Whew yes good catch (one more reason to have an independent client with message queue things :) ). How is this working like that though?

Reply:
Just curious, how come `GetAcquirableJobs` uses `c.ActionsServiceAdminToken` and `AcquireJobs` uses the message queue token?

Reply:
Everyone with `ActionsServiceAdminToken` can list acquirable jobs via `GetAcquirableJobs`, but only the active listener can `POST` to acquire the job via `AcquireJobs`

Reply:
Ah gotcha, thanks :)